### DISCUSSION TITLE
My Location Services Won't work.

### DISCUSSION THREAD

**Discussion by ExplosiveCat135** [community/community#35485] [GitHub Education]
Hello,
I was trying to apply for the Student Developer Pack but my location services would don't work. As you can see from the attached images, I have given both the GitHub website and Google Chrome access to my location. My location services would not work though, and this resulted in my application being rejected. If anyone could help me fix this I would be very appreciative. Thank you.
<img width="635" alt="Screenshot 2022-10-07 at 9 27 06 PM" src="https://user-images.githubusercontent.com/92578989/194677388-f57202b9-018a-411d-b50b-86f5744fa6cc.png">
<img width="497" alt="Screenshot 2022-10-07 at 9 27 20 PM" src="https://user-images.githubusercontent.com/92578989/194677408-5f0a6cda-4597-4db1-90bc-cd55ac22e473.png">
<img width="645" alt="Screenshot 2022-10-07 at 9 27 37 PM" src="https://user-images.githubusercontent.com/92578989/194677419-36363c0b-ba7d-480b-a0db-9c9c31f516c5.png">

**Comment by Saloh-Karimov** [comment]
Hi, I am having the same problem, have you managed to fix this?

**Comment by ExplosiveCat135** [comment]
No, I haven't. I am waiting for someone to help me. :(
